docs/Sprint-Review/sprint-101-summary.md

```diff
@@ -0,0 +1,87 @@
+# sprint-101-summary
+
+6/5/2024 - 6/18/2024
+
+**Dev:**
+
+_**Prioritized DAC and Notifications Work**_&#x20;
+
+* As sys admin, I want to be able to reparse datafile sets #2978
+* As a software engineer, I want to be able to test django-admin-508 #3008
+* As tech lead, I need the STT filter for search\_indexes to be updated #2950
+* As a data analyst I want to be notified of approaching data deadlines #2473
+* add `SENDGRID_API_KEY` to deploy.backend.sh #2677
+* Implement (small) data lifecycle (backup/archive ES) #3004
+* As a developer I want to test django-508 repo #2980\
+
+
+**DevOps:**
+
+_**Successful deployments across environments and pipeline stability investments**_
+
+* Application health monitoring #831
+
+**Design:**
+
+_**Close out error guide work, coordinate with dev on a plan for Cat 3 problems introduced by Cat 2 work, support spec-writing for upcoming work, and continued error audit dev ticket refinement.**_
+
+* Error Report Guide #2847 is going through final edits&#x20;
+  * Walk-on Dear Colleague letter link update to this PR (or spin up a separate ticket if deployment of the letter to OFA's website doesn't align to this)
+* Deliver spec for #3014 (Blanked-out values in Submission History)
+* \#3021 Updated KC Release Notes & Update Indicator FAQ - stretch goal for this sprint
+* Write follow-on / spec tickets from #2909 findings - stretch/ongoing lift
+* Category 3 error messages clean-up #2792 - stretch/ongoing lift
+
+## Tickets
+
+### Completed/Merged
+
+* [#2980 As a developer I want to test django-508 repo](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2980)
+* [#2892 Correct misleading error message for unaligned reporting year/q against header year/q](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2892)
```

Comment:
@robgendron @raftmsohani i think i missed why this has closed. its still an issue.

Reply:
since i raised this question, we decided to close #2892 and open a new bug ticket to address this @robgendron